pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -389,8 +389,8 @@ func emitSchedule(sj *jobs.ScheduledJob, backupStmt string, resultsCh chan<- tre
 	if sj.IsPaused() {
 		nextRun = tree.DNull
 		status = "PAUSED"
-		if reason := sj.LastChangeReason(); reason != "" {
-			status += ": " + reason
+		if s := sj.ScheduleStatus(); s != "" {
+			status += ": " + s
```

Comment:
I mention below that we shouldn't stick the time in `s` but here's another reason: this will read `state: time: message` now